DFC.ServiceTaxonomy.ApiFunction/OpenAPIDocs/GetAllOccupationsOpenApi.txt

```diff
@@ -0,0 +1,68 @@
+﻿openapi: 3.0.1
+info:
+  title: Get All Occupations
+  description: 'Sample Description for Query'
+  version: 1.0.0
+servers:
+- url: __FunctionAppUrl__
+tags:
+- name: Get All Occupations
+  description: Returns all occupations
+paths:
+  /Execute/:
+    get:
+      tags:
+      - Occupation
+      summary: Get all occupations.
+      description: Returns all occupations
+      operationId: GetAllOccupations
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Occupation'
+        204:
+          description: No content can be found.
+          content: {}
+        400:
+          description: Missing data from body or parameters have not been supplied for query.
+          content: {}
+        422:
+          description: Unprocessable Entity - Unable to read/deserialize data.
+          content: {}
+        500:
+          description: Missing App Settings or Config Files.
+          content: {}
+components:
+  schemas:
+    Occupation: 
+      type: "object"
+      properties:
+        occupations: 
+          type: array
+          items: 
+            type: object
+            properties: 
+              uri: 
+                type: string
+                example: http://data.europa.eu/esco/occupation/114e1eff-215e-47df-8e10-45a5b72f8197
+              occupation: 
+                type: string
+                example: renewable energy consultant
+              alternativeLabels: 
+                type: array
+                items: 
+                  type: string
+                  example: "alt 1"
+              lastmodified: 
+                type: string
+                example: 08/11/2019
```

Comment:
`lastmodified` values are returned in this format ... `2017-01-17T13:15:00Z`.
We might also want to check the required case of the key name. All other keys are camelCase, whereas lastmodified is all lower.